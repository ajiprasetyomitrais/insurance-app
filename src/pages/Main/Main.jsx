import { Container, Row, Col } from "react-bootstrap";
import Dashboard from '../Dashboard'
import Bell from "../../components/Icons/Bell";
import ChevronDown from "../../components/Icons/ChevronDown";
import Home from "../../components/Icons/Home";

const Main = () => {
  return (
    <Container fluid>
      <Row>
        <Col lg={2} data-testid="side_navigation" className="side-nav">
          <div data-testid="profile-user" className="profile-user">
            <img className="image-profile" src="images/elon.jpg" alt="" />
            <div className="details-profile">
              <p>
                Jhon Doe <ChevronDown />
              </p>
              <p>New York</p>
            </div>
          </div>
          <div className="group-nav">
            <div className="dashboard-nav">
              <Home />
              <span className="title-nav">Dashboard</span>
            </div>
          </div>
        </Col>
        <Col lg={10} className="right-content">
          <div className="top-nav">
            <h5>Prodigi</h5>
            <Bell />
          </div>
          <Dashboard />
        </Col>
      </Row>
    </Container>
  );
}

export default Main;
