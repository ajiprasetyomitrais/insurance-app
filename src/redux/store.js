import { combineReducers, configureStore } from "@reduxjs/toolkit";
import DashboardReducer from "../pages/Dashboard/DashboardSlice";

// export const rootReducer = combineReducers({
    
// });

export const store = configureStore({
    reducer: {
        DashboardReducer
    }
});

