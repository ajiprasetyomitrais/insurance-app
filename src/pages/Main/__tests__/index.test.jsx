import React from "react";
import renderer from "react-test-renderer";
import Main from "../Main";
import { store } from "../../../redux/store";
import { Provider } from "react-redux";
import { useNavigate } from "react-router-dom";

const mockedUsedNavigate = jest.fn();

jest.mock("react-router-dom", () => ({
  ...jest.requireActual("react-router-dom"),
  useNavigate: () => mockedUsedNavigate,
}));

describe("test main component", () => {
  it("should display main comoponent properly", () => {
    const tree = renderer.create(
      <Provider store={store}>
        <Main />
      </Provider>
    );
    expect(tree).toMatchSnapshot();
  });
});
