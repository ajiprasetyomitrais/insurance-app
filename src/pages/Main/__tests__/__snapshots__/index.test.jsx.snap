// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test main component should  1`] = `
<div
  className="container-fluid"
>
  <div
    className="row"
  >
    <div
      className="side-nav col-lg-2"
      data-testid="side_navigation"
    >
      <div
        className="profile-user"
        data-testid="profile-user"
      >
        <img
          alt=""
          className="image-profile"
          src="images/elon.jpg"
        />
        <div
          className="details-profile"
        >
          <p>
            Jhon Doe 
            <svg
              className="icon-style-white"
              viewBox="0 0 24 24"
            >
              <path
                d="M7.41,8.58L12,13.17L16.59,8.58L18,10L12,16L6,10L7.41,8.58Z"
                fill="currentColor"
              />
            </svg>
          </p>
          <p>
            New York
          </p>
        </div>
      </div>
      <div
        className="group-nav"
      >
        <div
          className="dashboard-nav"
        >
          <svg
            className="icon-style-white"
            viewBox="0 0 24 24"
          >
            <path
              d="M9,13H15V19H18V10L12,5.5L6,10V19H9V13M4,21V9L12,3L20,9V21H4Z"
              fill="currentColor"
            />
          </svg>
          <span
            className="title-nav"
          >
            Dashboard
          </span>
        </div>
      </div>
    </div>
    <div
      className="right-content col-lg-10"
    >
      <div
        className="top-nav"
      >
        <h5>
          Prodigi
        </h5>
        <svg
          className="icon-style"
          viewBox="0 0 24 24"
        >
          <path
            d="M10 21H14C14 22.1 13.1 23 12 23S10 22.1 10 21M21 19V20H3V19L5 17V11C5 7.9 7 5.2 10 4.3V4C10 2.9 10.9 2 12 2S14 2.9 14 4V4.3C17 5.2 19 7.9 19 11V17L21 19M17 11C17 8.2 14.8 6 12 6S7 8.2 7 11V18H17V11Z"
            fill="currentColor"
          />
        </svg>
      </div>
      <div>
        <div
          className="insurance-filter"
        >
          <div
            className="input-field"
          >
            <input
              className="form-control"
              name="searchInput"
              onChange={[Function]}
              placeholder="Search insurance..."
              type="text"
            />
          </div>
          <div
            className="filter-group"
          >
            <div
              className="dropdown"
            >
              <select
                className="select-type-dropdown"
                id="typeInsurance"
                name="typeInsurance"
                onChange={[Function]}
                value=""
              >
                <option
                  value="syariah"
                >
                  syariah
                </option>
                <option
                  value="non syariah"
                >
                  non syariah
                </option>
              </select>
            </div>
          </div>
        </div>
        <h5
          className="total-insurance"
        >
          8 Insurance
        </h5>
        <div
          className="list-insurance"
        >
          <div
            className="insurance-card"
          >
            <img
              alt="background-img"
              className="insurance-img"
              src="/images/bg_insurance.jpg"
            />
            <div
              className="insurance-details"
            >
              <h5
                className="insurance-title"
                onClick={[Function]}
              >
                Motor Insurance
              </h5>
              <p
                className="insurance-sub"
              >
                Traditional Life Insurance Protection until the age of 99 years with premium payment options as a valuable inheritance for loved ones.
              </p>
              <p
                className="insurance-price"
              >
                300000/month
              </p>
              <span
                className="insurance-type"
              >
                syariah
              </span>
            </div>
          </div>
          <div
            className="insurance-card"
          >
            <img
              alt="background-img"
              className="insurance-img"
              src="/images/bg_insurance.jpg"
            />
            <div
              className="insurance-details"
            >
              <h5
                className="insurance-title"
                onClick={[Function]}
              >
                Health Insurance
              </h5>
              <p
                className="insurance-sub"
              >
                Traditional Life Insurance Protection until the age of 99 years with premium payment options as a valuable inheritance for loved ones.
              </p>
              <p
                className="insurance-price"
              >
                500000/month
              </p>
              <span
                className="insurance-type"
              >
                non syariah
              </span>
            </div>
          </div>
          <div
            className="insurance-card"
          >
            <img
              alt="background-img"
              className="insurance-img"
              src="/images/bg_insurance.jpg"
            />
            <div
              className="insurance-details"
            >
              <h5
                className="insurance-title"
                onClick={[Function]}
              >
                Home Insurance
              </h5>
              <p
                className="insurance-sub"
              >
                Traditional Life Insurance Protection until the age of 99 years with premium payment options as a valuable inheritance for loved ones.
              </p>
              <p
                className="insurance-price"
              >
                200000/month
              </p>
              <span
                className="insurance-type"
              >
                syariah
              </span>
            </div>
          </div>
          <div
            className="insurance-card"
          >
            <img
              alt="background-img"
              className="insurance-img"
              src="/images/bg_insurance.jpg"
            />
            <div
              className="insurance-details"
            >
              <h5
                className="insurance-title"
                onClick={[Function]}
              >
                Health Insurance
              </h5>
              <p
                className="insurance-sub"
              >
                Traditional Life Insurance Protection until the age of 99 years with premium payment options as a valuable inheritance for loved ones.
              </p>
              <p
                className="insurance-price"
              >
                50000/month
              </p>
              <span
                className="insurance-type"
              >
                non syariah
              </span>
            </div>
          </div>
          <div
            className="insurance-card"
          >
            <img
              alt="background-img"
              className="insurance-img"
              src="/images/bg_insurance.jpg"
            />
            <div
              className="insurance-details"
            >
              <h5
                className="insurance-title"
                onClick={[Function]}
              >
                Travel Insurance
              </h5>
              <p
                className="insurance-sub"
              >
                Traditional Life Insurance Protection until the age of 99 years with premium payment options as a valuable inheritance for loved ones.
              </p>
              <p
                className="insurance-price"
              >
                1000000/month
              </p>
              <span
                className="insurance-type"
              >
                syariah
              </span>
            </div>
          </div>
          <div
            className="insurance-card"
          >
            <img
              alt="background-img"
              className="insurance-img"
              src="/images/bg_insurance.jpg"
            />
            <div
              className="insurance-details"
            >
              <h5
                className="insurance-title"
                onClick={[Function]}
              >
                Fire Insurance
              </h5>
              <p
                className="insurance-sub"
              >
                Traditional Life Insurance Protection until the age of 99 years with premium payment options as a valuable inheritance for loved ones.
              </p>
              <p
                className="insurance-price"
              >
                50000/month
              </p>
              <span
                className="insurance-type"
              >
                non syariah
              </span>
            </div>
          </div>
          <div
            className="insurance-card"
          >
            <img
              alt="background-img"
              className="insurance-img"
              src="/images/bg_insurance.jpg"
            />
            <div
              className="insurance-details"
            >
              <h5
                className="insurance-title"
                onClick={[Function]}
              >
                Home Insurance
              </h5>
              <p
                className="insurance-sub"
              >
                Traditional Life Insurance Protection until the age of 99 years with premium payment options as a valuable inheritance for loved ones.
              </p>
              <p
                className="insurance-price"
              >
                80000/month
              </p>
              <span
                className="insurance-type"
              >
                syariah
              </span>
            </div>
          </div>
          <div
            className="insurance-card"
          >
            <img
              alt="background-img"
              className="insurance-img"
              src="/images/bg_insurance.jpg"
            />
            <div
              className="insurance-details"
            >
              <h5
                className="insurance-title"
                onClick={[Function]}
              >
                Travel Insurance
              </h5>
              <p
                className="insurance-sub"
              >
                Traditional Life Insurance Protection until the age of 99 years with premium payment options as a valuable inheritance for loved ones.
              </p>
              <p
                className="insurance-price"
              >
                400000/month
              </p>
              <span
                className="insurance-type"
              >
                non syariah
              </span>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`test main component should display main comoponent properly  1`] = `
<div
  className="container-fluid"
>
  <div
    className="row"
  >
    <div
      className="side-nav col-lg-2"
      data-testid="side_navigation"
    >
      <div
        className="profile-user"
        data-testid="profile-user"
      >
        <img
          alt=""
          className="image-profile"
          src="images/elon.jpg"
        />
        <div
          className="details-profile"
        >
          <p>
            Jhon Doe 
            <svg
              className="icon-style-white"
              viewBox="0 0 24 24"
            >
              <path
                d="M7.41,8.58L12,13.17L16.59,8.58L18,10L12,16L6,10L7.41,8.58Z"
                fill="currentColor"
              />
            </svg>
          </p>
          <p>
            New York
          </p>
        </div>
      </div>
      <div
        className="group-nav"
      >
        <div
          className="dashboard-nav"
        >
          <svg
            className="icon-style-white"
            viewBox="0 0 24 24"
          >
            <path
              d="M9,13H15V19H18V10L12,5.5L6,10V19H9V13M4,21V9L12,3L20,9V21H4Z"
              fill="currentColor"
            />
          </svg>
          <span
            className="title-nav"
          >
            Dashboard
          </span>
        </div>
      </div>
    </div>
    <div
      className="right-content col-lg-10"
    >
      <div
        className="top-nav"
      >
        <h5>
          Prodigi
        </h5>
        <svg
          className="icon-style"
          viewBox="0 0 24 24"
        >
          <path
            d="M10 21H14C14 22.1 13.1 23 12 23S10 22.1 10 21M21 19V20H3V19L5 17V11C5 7.9 7 5.2 10 4.3V4C10 2.9 10.9 2 12 2S14 2.9 14 4V4.3C17 5.2 19 7.9 19 11V17L21 19M17 11C17 8.2 14.8 6 12 6S7 8.2 7 11V18H17V11Z"
            fill="currentColor"
          />
        </svg>
      </div>
      <div>
        <div
          className="insurance-filter"
        >
          <div
            className="input-field"
          >
            <input
              className="form-control"
              name="searchInput"
              onChange={[Function]}
              placeholder="Search insurance..."
              type="text"
            />
          </div>
          <div
            className="filter-group"
          >
            <div
              className="dropdown"
            >
              <select
                className="select-type-dropdown"
                id="typeInsurance"
                name="typeInsurance"
                onChange={[Function]}
                value=""
              >
                <option
                  value="syariah"
                >
                  syariah
                </option>
                <option
                  value="non syariah"
                >
                  non syariah
                </option>
              </select>
            </div>
          </div>
        </div>
        <h5
          className="total-insurance"
        >
          8 Insurance
        </h5>
        <div
          className="list-insurance"
        >
          <div
            className="insurance-card"
          >
            <img
              alt="background-img"
              className="insurance-img"
              src="/images/bg_insurance.jpg"
            />
            <div
              className="insurance-details"
            >
              <h5
                className="insurance-title"
                onClick={[Function]}
              >
                Motor Insurance
              </h5>
              <p
                className="insurance-sub"
              >
                Traditional Life Insurance Protection until the age of 99 years with premium payment options as a valuable inheritance for loved ones.
              </p>
              <p
                className="insurance-price"
              >
                300000/month
              </p>
              <span
                className="insurance-type"
              >
                syariah
              </span>
            </div>
          </div>
          <div
            className="insurance-card"
          >
            <img
              alt="background-img"
              className="insurance-img"
              src="/images/bg_insurance.jpg"
            />
            <div
              className="insurance-details"
            >
              <h5
                className="insurance-title"
                onClick={[Function]}
              >
                Health Insurance
              </h5>
              <p
                className="insurance-sub"
              >
                Traditional Life Insurance Protection until the age of 99 years with premium payment options as a valuable inheritance for loved ones.
              </p>
              <p
                className="insurance-price"
              >
                500000/month
              </p>
              <span
                className="insurance-type"
              >
                non syariah
              </span>
            </div>
          </div>
          <div
            className="insurance-card"
          >
            <img
              alt="background-img"
              className="insurance-img"
              src="/images/bg_insurance.jpg"
            />
            <div
              className="insurance-details"
            >
              <h5
                className="insurance-title"
                onClick={[Function]}
              >
                Home Insurance
              </h5>
              <p
                className="insurance-sub"
              >
                Traditional Life Insurance Protection until the age of 99 years with premium payment options as a valuable inheritance for loved ones.
              </p>
              <p
                className="insurance-price"
              >
                200000/month
              </p>
              <span
                className="insurance-type"
              >
                syariah
              </span>
            </div>
          </div>
          <div
            className="insurance-card"
          >
            <img
              alt="background-img"
              className="insurance-img"
              src="/images/bg_insurance.jpg"
            />
            <div
              className="insurance-details"
            >
              <h5
                className="insurance-title"
                onClick={[Function]}
              >
                Health Insurance
              </h5>
              <p
                className="insurance-sub"
              >
                Traditional Life Insurance Protection until the age of 99 years with premium payment options as a valuable inheritance for loved ones.
              </p>
              <p
                className="insurance-price"
              >
                50000/month
              </p>
              <span
                className="insurance-type"
              >
                non syariah
              </span>
            </div>
          </div>
          <div
            className="insurance-card"
          >
            <img
              alt="background-img"
              className="insurance-img"
              src="/images/bg_insurance.jpg"
            />
            <div
              className="insurance-details"
            >
              <h5
                className="insurance-title"
                onClick={[Function]}
              >
                Travel Insurance
              </h5>
              <p
                className="insurance-sub"
              >
                Traditional Life Insurance Protection until the age of 99 years with premium payment options as a valuable inheritance for loved ones.
              </p>
              <p
                className="insurance-price"
              >
                1000000/month
              </p>
              <span
                className="insurance-type"
              >
                syariah
              </span>
            </div>
          </div>
          <div
            className="insurance-card"
          >
            <img
              alt="background-img"
              className="insurance-img"
              src="/images/bg_insurance.jpg"
            />
            <div
              className="insurance-details"
            >
              <h5
                className="insurance-title"
                onClick={[Function]}
              >
                Fire Insurance
              </h5>
              <p
                className="insurance-sub"
              >
                Traditional Life Insurance Protection until the age of 99 years with premium payment options as a valuable inheritance for loved ones.
              </p>
              <p
                className="insurance-price"
              >
                50000/month
              </p>
              <span
                className="insurance-type"
              >
                non syariah
              </span>
            </div>
          </div>
          <div
            className="insurance-card"
          >
            <img
              alt="background-img"
              className="insurance-img"
              src="/images/bg_insurance.jpg"
            />
            <div
              className="insurance-details"
            >
              <h5
                className="insurance-title"
                onClick={[Function]}
              >
                Home Insurance
              </h5>
              <p
                className="insurance-sub"
              >
                Traditional Life Insurance Protection until the age of 99 years with premium payment options as a valuable inheritance for loved ones.
              </p>
              <p
                className="insurance-price"
              >
                80000/month
              </p>
              <span
                className="insurance-type"
              >
                syariah
              </span>
            </div>
          </div>
          <div
            className="insurance-card"
          >
            <img
              alt="background-img"
              className="insurance-img"
              src="/images/bg_insurance.jpg"
            />
            <div
              className="insurance-details"
            >
              <h5
                className="insurance-title"
                onClick={[Function]}
              >
                Travel Insurance
              </h5>
              <p
                className="insurance-sub"
              >
                Traditional Life Insurance Protection until the age of 99 years with premium payment options as a valuable inheritance for loved ones.
              </p>
              <p
                className="insurance-price"
              >
                400000/month
              </p>
              <span
                className="insurance-type"
              >
                non syariah
              </span>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`test main component should display main comoponent properly 1`] = `
<div
  className="container-fluid"
>
  <div
    className="row"
  >
    <div
      className="side-nav col-lg-2"
      data-testid="side_navigation"
    >
      <div
        className="profile-user"
        data-testid="profile-user"
      >
        <img
          alt=""
          className="image-profile"
          src="images/elon.jpg"
        />
        <div
          className="details-profile"
        >
          <p>
            Jhon Doe 
            <svg
              className="icon-style-white"
              viewBox="0 0 24 24"
            >
              <path
                d="M7.41,8.58L12,13.17L16.59,8.58L18,10L12,16L6,10L7.41,8.58Z"
                fill="currentColor"
              />
            </svg>
          </p>
          <p>
            New York
          </p>
        </div>
      </div>
      <div
        className="group-nav"
      >
        <div
          className="dashboard-nav"
        >
          <svg
            className="icon-style-white"
            viewBox="0 0 24 24"
          >
            <path
              d="M9,13H15V19H18V10L12,5.5L6,10V19H9V13M4,21V9L12,3L20,9V21H4Z"
              fill="currentColor"
            />
          </svg>
          <span
            className="title-nav"
          >
            Dashboard
          </span>
        </div>
      </div>
    </div>
    <div
      className="right-content col-lg-10"
    >
      <div
        className="top-nav"
      >
        <h5>
          Prodigi
        </h5>
        <svg
          className="icon-style"
          viewBox="0 0 24 24"
        >
          <path
            d="M10 21H14C14 22.1 13.1 23 12 23S10 22.1 10 21M21 19V20H3V19L5 17V11C5 7.9 7 5.2 10 4.3V4C10 2.9 10.9 2 12 2S14 2.9 14 4V4.3C17 5.2 19 7.9 19 11V17L21 19M17 11C17 8.2 14.8 6 12 6S7 8.2 7 11V18H17V11Z"
            fill="currentColor"
          />
        </svg>
      </div>
      <div>
        <div
          className="insurance-filter"
        >
          <div
            className="input-field"
          >
            <input
              className="form-control"
              name="searchInput"
              onChange={[Function]}
              placeholder="Search insurance..."
              type="text"
            />
          </div>
          <div
            className="filter-group"
          >
            <div
              className="dropdown"
            >
              <select
                className="select-type-dropdown"
                id="typeInsurance"
                name="typeInsurance"
                onChange={[Function]}
                value=""
              >
                <option
                  value="syariah"
                >
                  syariah
                </option>
                <option
                  value="non syariah"
                >
                  non syariah
                </option>
              </select>
            </div>
          </div>
        </div>
        <h5
          className="total-insurance"
        >
          8 Insurance
        </h5>
        <div
          className="list-insurance"
        >
          <div
            className="insurance-card"
          >
            <img
              alt="background-img"
              className="insurance-img"
              src="/images/bg_insurance.jpg"
            />
            <div
              className="insurance-details"
            >
              <h5
                className="insurance-title"
                onClick={[Function]}
              >
                Motor Insurance
              </h5>
              <p
                className="insurance-sub"
              >
                Traditional Life Insurance Protection until the age of 99 years with premium payment options as a valuable inheritance for loved ones.
              </p>
              <p
                className="insurance-price"
              >
                300000/month
              </p>
              <span
                className="insurance-type"
              >
                syariah
              </span>
            </div>
          </div>
          <div
            className="insurance-card"
          >
            <img
              alt="background-img"
              className="insurance-img"
              src="/images/bg_insurance.jpg"
            />
            <div
              className="insurance-details"
            >
              <h5
                className="insurance-title"
                onClick={[Function]}
              >
                Health Insurance
              </h5>
              <p
                className="insurance-sub"
              >
                Traditional Life Insurance Protection until the age of 99 years with premium payment options as a valuable inheritance for loved ones.
              </p>
              <p
                className="insurance-price"
              >
                500000/month
              </p>
              <span
                className="insurance-type"
              >
                non syariah
              </span>
            </div>
          </div>
          <div
            className="insurance-card"
          >
            <img
              alt="background-img"
              className="insurance-img"
              src="/images/bg_insurance.jpg"
            />
            <div
              className="insurance-details"
            >
              <h5
                className="insurance-title"
                onClick={[Function]}
              >
                Home Insurance
              </h5>
              <p
                className="insurance-sub"
              >
                Traditional Life Insurance Protection until the age of 99 years with premium payment options as a valuable inheritance for loved ones.
              </p>
              <p
                className="insurance-price"
              >
                200000/month
              </p>
              <span
                className="insurance-type"
              >
                syariah
              </span>
            </div>
          </div>
          <div
            className="insurance-card"
          >
            <img
              alt="background-img"
              className="insurance-img"
              src="/images/bg_insurance.jpg"
            />
            <div
              className="insurance-details"
            >
              <h5
                className="insurance-title"
                onClick={[Function]}
              >
                Health Insurance
              </h5>
              <p
                className="insurance-sub"
              >
                Traditional Life Insurance Protection until the age of 99 years with premium payment options as a valuable inheritance for loved ones.
              </p>
              <p
                className="insurance-price"
              >
                50000/month
              </p>
              <span
                className="insurance-type"
              >
                non syariah
              </span>
            </div>
          </div>
          <div
            className="insurance-card"
          >
            <img
              alt="background-img"
              className="insurance-img"
              src="/images/bg_insurance.jpg"
            />
            <div
              className="insurance-details"
            >
              <h5
                className="insurance-title"
                onClick={[Function]}
              >
                Travel Insurance
              </h5>
              <p
                className="insurance-sub"
              >
                Traditional Life Insurance Protection until the age of 99 years with premium payment options as a valuable inheritance for loved ones.
              </p>
              <p
                className="insurance-price"
              >
                1000000/month
              </p>
              <span
                className="insurance-type"
              >
                syariah
              </span>
            </div>
          </div>
          <div
            className="insurance-card"
          >
            <img
              alt="background-img"
              className="insurance-img"
              src="/images/bg_insurance.jpg"
            />
            <div
              className="insurance-details"
            >
              <h5
                className="insurance-title"
                onClick={[Function]}
              >
                Fire Insurance
              </h5>
              <p
                className="insurance-sub"
              >
                Traditional Life Insurance Protection until the age of 99 years with premium payment options as a valuable inheritance for loved ones.
              </p>
              <p
                className="insurance-price"
              >
                50000/month
              </p>
              <span
                className="insurance-type"
              >
                non syariah
              </span>
            </div>
          </div>
          <div
            className="insurance-card"
          >
            <img
              alt="background-img"
              className="insurance-img"
              src="/images/bg_insurance.jpg"
            />
            <div
              className="insurance-details"
            >
              <h5
                className="insurance-title"
                onClick={[Function]}
              >
                Home Insurance
              </h5>
              <p
                className="insurance-sub"
              >
                Traditional Life Insurance Protection until the age of 99 years with premium payment options as a valuable inheritance for loved ones.
              </p>
              <p
                className="insurance-price"
              >
                80000/month
              </p>
              <span
                className="insurance-type"
              >
                syariah
              </span>
            </div>
          </div>
          <div
            className="insurance-card"
          >
            <img
              alt="background-img"
              className="insurance-img"
              src="/images/bg_insurance.jpg"
            />
            <div
              className="insurance-details"
            >
              <h5
                className="insurance-title"
                onClick={[Function]}
              >
                Travel Insurance
              </h5>
              <p
                className="insurance-sub"
              >
                Traditional Life Insurance Protection until the age of 99 years with premium payment options as a valuable inheritance for loved ones.
              </p>
              <p
                className="insurance-price"
              >
                400000/month
              </p>
              <span
                className="insurance-type"
              >
                non syariah
              </span>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
